frontend/src/fetchers/citations.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable import/prefer-default-export */
+import join from 'url-join';
+import {
+  get,
+} from './index';
+
+export async function fetchCitationsByGrant(region, grantIds, reportStartDate) {
+  const formattedDate = new Date(reportStartDate).toISOString().split('T')[0];
```

Comment:
are we formatting this into the format it was already in?

Reply:
We need YYYY-MM-DD see before and after:

![image](https://github.com/user-attachments/assets/79384a94-7d7d-43e0-85c5-862667eeb5d1)

Reply:
oh ok